dimod/constrained.py

```diff
@@ -95,45 +95,45 @@ class ConstrainedQuadraticModel:
 
     Example:
 
-        Solve a simple `bin packing problem <https://w.wiki/3jz4>`_. In this
-        problem we wish to pack a set of items of different weights into
-        the smallest number of bins possible.
+        This example solves the simple `bin packing problem <https://w.wiki/3jz4>`_
+        problem of packing a set of items of different weights into the smallest
```

Comment:
Won't this read `This examples solves the simple bin packing problem problem of packing a set...`